mix.exs

```diff
@@ -50,10 +51,11 @@ defmodule Membrane.Element.UDP.MixProject do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.8.0"},
+      {:membrane_core, "~> 0.9.0"},
       {:ex_doc, "~> 0.21", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 1.0.0", only: [:dev, :test], runtime: false},
```

Comment:
These aren't needed in `:test` env.
Dialyxir should be updated & have the minor unlocked
```suggestion
      {:ex_doc, "~> 0.28", only: :dev, runtime: false},
      {:dialyxir, "~> 1.1", only: :dev, runtime: false},
```